src/services/Store/store/persist.config.ts

```diff
@@ -168,14 +170,31 @@ export const migrations = {
       ...state,
       settings: dissoc('inactivityTimer', state.settings)
     };
+  },
+  6: (state: DataStore) => {
+    return {
+      ...state,
+      // @ts-expect-error dPath is present on data to be migrated, want to remove it
```

Comment:
Why?

Reply:
Because I removed it from the type so we don't expect it to be there in the future.